deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -15,8 +15,16 @@
             "host": {
                 "sourcePath": "/media/nvme0n1/pgdata"
             }
+        }, {
+            "name": "tmp",
+            "host": {
+                "sourcePath": "/media/nvme0n1/tmp"
+            }
         }],
         "environment": [{
+            "name": "NB_MAX_TRIP_DISTANCE",
+            "value": "2680"
```

Comment:
Ah, yes, wasn't implying your solution wouldn't work, because it would, and is definitely faster to implement. Was just suggesting that while I'm dabbling here its worth the extra few minutes to define these values farther up to make explicit an assumption we're currently making (that these values are the same across the analysis)